src/modules/site-v2/templates/tools/heritability_calculator/result.html

```diff
@@ -209,14 +210,26 @@
             for (var cc=0; cc<UniqueNames.length; cc++){cpal.push(getRandomColor())};
         }
 
-        width = document.getElementById("svgchartarea").offsetWidth - 100; /* 100px is from the chart legend*/
-        height = 9 * width / 16;
-        console.log(width)
-        var chart = d3.exploding_boxplot(df, {y:'Value', group:'Strain', color:'AssayNumber', label:'TraitName', pal:cpal})
-                      .width(width)
-                      .height(height);
-        document.getElementById("svgchartarea").innerHTML = "";
-        chart('#svgchartarea');
+        // Create & save a function to compute the chart size & re-render it
+        renderPlot = function() {
```

Comment:
when declaring JS vars in new code, prefer `let` and `const`. For lambda-type functions, use `const`

Reply:
consider `const renderPlot = ...`

Reply:
In this particular case, I'm actually re-assigning the value of `renderPlot` (declared on line 207) to make it available outside the inner function context.  (Not the best solution, but it involved the least code restructuring.)

For the top-level declaration of `renderPlot`, updated to `let` in 94bdca02fdd6f97a4d5ba2b37220d4be65abe2a8